framework/assets/yii.js

```diff
@@ -143,16 +143,49 @@ yii = (function ($) {
          *
          * @param $e the jQuery representation of the element
          */
-        handleAction: function ($e) {
+        handleAction: function ($e,event) {
             var method = $e.data('method'),
                 $form = $e.closest('form'),
                 action = $e.attr('href'),
-                params = $e.data('params');
+                params = $e.data('params'),
+                pjax = $e.data('pjax'),
+                pjaxPushState = $e.data('pjax-push-state')? true : false,
```

Comment:
Casting to boolean could be done using `!!` i.e. `!!$e.data('pjax-push-state')`.